build/common.gypi

```diff
@@ -20,6 +21,9 @@
       ['enable_murphy==1', {
         'defines': ['ENABLE_MURPHY=1'],
       }],
+      ['OS=="android" and enable_webcl==1', {
```

Comment:
I think it makes more sense to separate these: here, you only check if `enable_webcl==1`, and before this you set `enable_webcl%` to 0 or 1 depending on whether you're on Android or not.